Includes/Rosetta/Tasks/SimpleTasks/GetPlayerGameTagTask.hpp

```diff
@@ -0,0 +1,39 @@
+// This code is based on Sabberstone project.
+// Copyright (c) 2017-2019 SabberStone Team, darkfriend77 & rnilva
+// RosettaStone is hearthstone simulator using C++ with reinforcement learning.
+// Copyright (c) 2019 Chris Ohk, Youngjoong Kim, SeungHyun Jeon
+
+#ifndef ROSETTASTONE_GET_PLAYER_GAME_TAG_TASK_HPP
+#define ROSETTASTONE_GET_PLAYER_GAME_TAG_TASK_HPP
+
+#include <Rosetta/Tasks/ITask.hpp>
+
+namespace RosettaStone::SimpleTasks
+{
+//!
+//! \brief GetGameTagTask class.
+//!
+//! This class represents the task for getting game tag from player.
+//!
+class GetPlayerGameTagTask : public ITask
+{
+ public:
+    //! Constructs task with given \p tag.
+    //! \param tag A game tag to get.
+    GetPlayerGameTagTask(GameTag tag);
+
+ private:
+    //! Processes task logic internally and returns meta data.
+    //! \param player The player to run task.
+    //! \return The result of task processing.
+    TaskStatus Impl(Player* player) override;
+
+    //! Internal method of Clone().
+    //! \return The cloned task.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class 'GetPlayerGameTagTask' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/app/utilForever/RosettaStone/pullRequest?prid=4719547)